Bio/motifs/transfac.py

```diff
@@ -193,24 +183,20 @@ def read(handle, strict=True):
             index, separator, accession = value.partition(";")
             if index[0] != "[":
                 raise ValueError(
-                    'The index "{0:s}" in a TRANSFAC RN line '
-                    "should start with a "
-                    '"[": "{0:s}"'.format(index, line)
+                    f'The index "{index}" in a TRANSFAC RN line should start'
+                    f' with a "[": "{line}"'
                 )
             if index[-1] != "]":
                 raise ValueError(
-                    'The index "{0:s}" in a TRANSFAC RN line '
-                    "should end with a "
-                    '"]": "{0:s}"'.format(index, line)
```

Comment:
This example had a bug, ``{0:s}`` instead of ``{1:s}`` - a perfect example of a downside of this syntax.